src/components/shared/DetailsWidget/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import ColorTag from '~shared/ColorTag';
+import { Domain } from '~gql';
```

Comment:
Shouldn't this be `Domain` (as in the fragment we're querying from the API) exported from `~types`?

Reply:
:point_up: